code/game/machinery/pipe/construction.dm

```diff
@@ -18,11 +18,16 @@ Buildable meters
 	icon_state = "simple"
 	inhand_icon_state = "buildpipe"
 	w_class = WEIGHT_CLASS_NORMAL
+	///Piping layer that we are going to be on
 	var/piping_layer = PIPING_LAYER_DEFAULT
+	///Type of pipe-object made, selected from the RPD
 	var/RPD_type
-	/// Whether it can be painted
+	///Whether it can be painted
 	var/paintable = FALSE
+	///Color of the pipe is going to be made from this pipe-object
 	var/pipe_color
+	///Initial direction of the created pipe (either made from the RPD or after unwrenching the pipe)
+	var/p_init_dir = 0
```

Comment:
What does `0` mean in this case? Smart? If so, it should have its own named define, as `0` is not a valid direction as far as I know.

Reply:
ah i should've call it SOUTH, it gets overridden anyway

Reply:
Use `SOUTH` everywhere else then.